tensorflow/1.10/README.md

```diff
@@ -0,0 +1,364 @@
+It also comes with instructions on how to deploy your own examples.
+
+
+<a name="uninstalling"></a>
+# Uninstalling
+
+See the uninstallation documentation [here](../uninstall/README.md)
+
+<a name="connecting-clients"></a>
+# Connecting Clients
+
+One of the benefits of running containerized services is that they can be placed anywhere in the cluster.
+Because they can be deployed anywhere on the cluster, clients need a way to find the service. This is where service discovery comes in.
+
+<a name="discovering-endpoints"></a>
+## Discovering Endpoints
+
+Once the service is running, you may view information about its endpoints via either of the following methods:
+- CLI:
+  - List endpoint types: `dcos beta-tensorflow --name=/tensorflow endpoints`
+  - View endpoints for an endpoint type: `dcos beta-tensorflow endpoints <endpoint>`
+- Web:
+  - List endpoint types: `<dcos-url>/service/tensorflow/v1/endpoints`
+  - View endpoints for an endpoint type: `<dcos-url>/service/tensorflow/v1/endpoints/<endpoint>`
+
+Returned endpoints will include the following:
+- `.autoip.dcos.thisdcos.directory` hostnames for each instance that will follow them if they're moved within the DC/OS cluster.
+- A HA-enabled VIP hostname for accessing any of the instances (optional).
+- A direct IP address for accesssing the service if `.autoip.dcos.thisdcos.directory` hostnames are not resolvable.
+- If your service is on the `dcos` overlay network, then the IP will be from the subnet allocated to the host that the task is running on.
+It will not be the host IP. To resolve the host IP use Mesos DNS (`<task>.<service>.mesos`).
+
+In general, the `.autoip.dcos.thisdcos.directory` and `<task>.<service>.mesos` endpoints will only work from within the same DC/OS cluster.
+From outside the cluster you can either use the direct IPs or set up a proxy service that acts as a frontend to your TensorFlow instance.
+For development and testing purposes, you can use [DC/OS Tunnel](https://docs.mesosphere.com/latest/administration/access-node/tunnel/)
+to access services from outside the cluster, but this option is not suitable for production use.
+
+<a name="managing"></a>
+# Managing
+
+<a name="updating-configuration"></a>
+## Updating Configuration
+You can make changes to the service after it has been launched.
+Configuration management is handled by the scheduler process, which in turn handles deploying TensorFlow itself.
+
+Edit the runtime environment of the scheduler to make configuration changes.
+After making a change, the scheduler will be restarted and automatically deploy any detected changes to the service, one node at a time.
+For example, a given change will first be applied to `worker-0`, then `worker-1`, and so on.
+
+Some changes, such as decreasing the number of nodes or changing volume requirements, are not supported after initial deployment.
+See [Limitations](#limitations).
```

Comment:
This should probably be a link over to the TF docs as well.  However, that section seems to be missing...

https://github.com/mesosphere/dcos-tensorflow/blob/master/docs/index.md#limitations

/cc @elezar